dayrepo/snippets/static/js/process_add.js

```diff
@@ -0,0 +1,9 @@
+var i = 1 ;
+function addForm() {
+  var content_area = document.getElementById("form_area");
```

Comment:
```suggestion
  let content_area = document.getElementById("form_area");
```
これも上記と同じ理由です。
また、基本的には const で宣言する。再代入したくなったら let に変える。という意識で実装するといいと言われています。今回はこの下で変数に変更を加えているため、たぶん let っぽいです。
https://qiita.com/kerupani129/items/c56ae227b8ff6f026884
> プログラミング全般において、変更可能な (ミュータブルな) 変数よりも変更不可な (イミュータブルな) 変数の方が望ましいです。可読性・保守性の向上、バグ削減などのメリットがあります。